src/main/java/uk/gov/hmcts/reform/sscs/service/VenueDataLoader.java

```diff
@@ -27,6 +28,7 @@ public class VenueDataLoader {
     private final Map<String, VenueDetails> venueDetailsMapByVenueName = newHashMap();
     private final Map<String, VenueDetails> activeVenueDetailsMapByEpimsId = newHashMap();
     private final Map<String, VenueDetails> activeVenueDetailsMapByPostcode = newHashMap();
+    private final Map<RegionalProcessingCenter, VenueDetails> venueDetailsMapByRpc = newHashMap();
```

Comment:
If you are only interested in the RPC name, I would use 
```suggestion
    private final Map<String, VenueDetails> venueDetailsMapByRpc = newHashMap();
```
This should be more efficient and less work to read from.

Reply:
Yeah I was going off some pointers from Del re: mapping RPC -> Venues, taking that to mean using the objects for RPC and VenueDetails. Guessing I've misunderstood slightly what he wanted as thought it was about holding that info in case we needed it in future too.

As we'd encounter duplicate key issues if RPC name is used ("SSCS Leeds" for example), might be worth doing <String, List<VenueDetails>> and collecting the venue details as a list for each RPC name key. Then, that makes the incoming get from the service class a touch different too. Alternative is switch the key to be the venue details or venue epimsid.